2023-09-28 22:16:48,659 INFO    MainThread:51940 [wandb_setup.py:_flush():76] Current SDK version is 0.15.11
2023-09-28 22:16:48,659 INFO    MainThread:51940 [wandb_setup.py:_flush():76] Configure stats pid to 51940
2023-09-28 22:16:48,659 INFO    MainThread:51940 [wandb_setup.py:_flush():76] Loading settings from /Users/oliverclive-griffin/.config/wandb/settings
2023-09-28 22:16:48,659 INFO    MainThread:51940 [wandb_setup.py:_flush():76] Loading settings from /Users/oliverclive-griffin/personal/transformer-from-scratch/wandb/settings
2023-09-28 22:16:48,659 INFO    MainThread:51940 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-09-28 22:16:48,659 INFO    MainThread:51940 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-09-28 22:16:48,659 INFO    MainThread:51940 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'train.py', 'program_abspath': '/Users/oliverclive-griffin/personal/transformer-from-scratch/train.py', 'program': '/Users/oliverclive-griffin/personal/transformer-from-scratch/train.py'}
2023-09-28 22:16:48,659 INFO    MainThread:51940 [wandb_init.py:_log_setup():528] Logging user logs to /Users/oliverclive-griffin/personal/transformer-from-scratch/wandb/run-20230928_221648-pey4lphl/logs/debug.log
2023-09-28 22:16:48,659 INFO    MainThread:51940 [wandb_init.py:_log_setup():529] Logging internal logs to /Users/oliverclive-griffin/personal/transformer-from-scratch/wandb/run-20230928_221648-pey4lphl/logs/debug-internal.log
2023-09-28 22:16:48,659 INFO    MainThread:51940 [wandb_init.py:init():568] calling init triggers
2023-09-28 22:16:48,659 INFO    MainThread:51940 [wandb_init.py:init():575] wandb.init called with sweep_config: {}
config: {'batch_size': 32, 'd_in': 128, 'block_size': 64, 'train_pct': 0.8, 'num_block': 6, 'num_heads': 4, 'batches': 435}
2023-09-28 22:16:48,659 INFO    MainThread:51940 [wandb_init.py:init():618] starting backend
2023-09-28 22:16:48,659 INFO    MainThread:51940 [wandb_init.py:init():622] setting up manager
2023-09-28 22:16:48,663 INFO    MainThread:51940 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2023-09-28 22:16:48,664 INFO    MainThread:51940 [wandb_init.py:init():628] backend started and connected
2023-09-28 22:16:48,670 INFO    MainThread:51940 [wandb_init.py:init():720] updated telemetry
2023-09-28 22:16:48,682 INFO    MainThread:51940 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2023-09-28 22:16:49,275 ERROR   MainThread:51940 [wandb_init.py:init():779] encountered error: It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 403: Forbidden)
2023-09-28 22:16:49,452 ERROR   MainThread:51940 [wandb_init.py:init():1188] It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 403: Forbidden)
Traceback (most recent call last):
  File "/Users/oliverclive-griffin/.local/share/virtualenvs/transformer-from-scratch-IzBRPy8T/lib/python3.11/site-packages/wandb/sdk/wandb_init.py", line 1170, in init
    run = wi.init()
          ^^^^^^^^^
  File "/Users/oliverclive-griffin/.local/share/virtualenvs/transformer-from-scratch-IzBRPy8T/lib/python3.11/site-packages/wandb/sdk/wandb_init.py", line 785, in init
    raise error
wandb.errors.CommError: It appears that you do not have permission to access the requested resource. Please reach out to the project owner to grant you access. If you have the correct permissions, verify that there are no issues with your networking setup.(Error 403: Forbidden)
2023-09-28 22:16:50,462 WARNING MsgRouterThr:51940 [router.py:message_loop():77] message_loop has been closed
